apiserver/facades/client/secrets/register.go

```diff
@@ -59,6 +59,7 @@ func newSecretsAPI(context facade.ModelContext) (*SecretsAPI, error) {
 		authTag := model.ModelTag()
 		return secrets.BackendConfigInfo(
```

Comment:
Just call the secretbackend service now.

Reply:
I've added a TODO - the backend service wasn't ready initially.
So can we land this and then followup with the next PR. We also need to deal with the use of the admin config getter as well now that the backend service is landed.

Reply:
Sorry, This is actually what I'm doing now.